src/CSF.SRDashboard.Client/Models/FullName.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Models
+{
+    public class FullName
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+
+        string _names;
+        public string Names
```

Comment:
The reason I suggest changing the variable name is most of the time when you have a plural it indicates a list value type and in this case it does not.
```suggestion
        public string FullName
```